install/automationservicebroker/deploy-automation-broker-apb.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: automation-broker-apb
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: automation-broker-apb
+    namespace: "${NAMESPACE}"
+
+- apiVersion: rbac.authorization.k8s.io/v1beta1
+  kind: ClusterRoleBinding
+  metadata:
+    name: automation-broker-apb
+  roleRef:
+    name: cluster-admin
+    kind: ClusterRole
+    apiGroup: rbac.authorization.k8s.io
+  subjects:
+  - kind: ServiceAccount
+    name: automation-broker-apb
+    namespace: "${NAMESPACE}"
+
+- apiVersion: batch/v1
+  kind: Job
+  metadata:
+    name: automation-broker-apb
+  spec:
+    backoffLimit: 5
+    activeDeadlineSeconds: 300
+    template:
+      spec:
+        restartPolicy: OnFailure
+        serviceAccount: automation-broker-apb
+        containers:
+        - image: docker.io/automationbroker/automation-broker-apb:latest
+          name: automation-broker-apb
+          args: [ "provision", "-e", "broker_name=automation-service-broker", "-e", "create_broker_namespace=true" ]
+
+parameters:
+- description: Namespace of the project that is being deploy
+  displayname: broker client cert key
+  name: NAMESPACE
+  value: "automation-broker-apb"
```

Comment:
We reserve `openshift-***`.  You should use a namespace under there.

@openshift/api-review this is our first external-ish component.  How shall we reserve names?